uber/templates/regform.html

```diff
@@ -468,7 +468,7 @@ <h4 class="h4-inline">
     <div class="col-sm-6">
         <input type="text" name="cellphone" id="cellphone" value="{{ attendee.cellphone }}" class="form-control" placeholder="Your Personal Cellphone Number">
     </div>
-    {% if not attendee.is_dealer %}
+    {% if  c.PAGE_PATH == '/registration/form' %}
```

Comment:
This will ONLY show the checkbox on the admin page for all attendees, so no one will be able to opt out of providing a cellphone number. We only want to prevent Dealers from opting out.

The full check you want is:
`{% if not attendee.is_dealer or c.PAGE_PATH == '/registration/form' %}`

This will show the field for both logged-in admins and non-dealers.